src/figuresClasses.test.ts

```diff
@@ -116,7 +116,7 @@ describe('Method "getArea" of class "Circle"', () => {
     const circle = new Circle('red', 6);
 
     expect(circle.getArea())
-      .toBe(113.09);
+      .toBe(113.10);
   });
```

Comment:
Well, nice try 😅
have you tried other ways to round number?

Reply:
totally not agry! because Before round it's square is 113.097 which is equal 113.1!
"returns the area of the figure rounded down to hundredths"

It's a mistake in test

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed

Reply:
Okay, good point but there is a purpose in that `down` word of this sentence 🙂 

Here is a good sample of differences, check this link

https://www.quora.com/Whats-the-difference-between-round-up-round-down-and-round-off

Reply:
Still, if you not agree with this suggestion, you may ask opinion from other mentors/coaches 🙂  I can be wrong

Reply:
Lets open wikipedia https://ru.wikipedia.org/wiki/%D0%9E%D0%BA%D1%80%D1%83%D0%B3%D0%BB%D0%B5%D0%BD%D0%B8%D0%B5
May be this task was created when method toFix wasn't implemented, so we have a tool to calculate properly.

Reply:
**rounded down**.

sorry mate, I was sure that this one is general round